Flick/Controllers/ListViewController.swift

```diff
@@ -34,21 +34,26 @@ class ListViewController: UIViewController {
     // MARK: - Private Data Vars
     private let cellPadding: CGFloat = 20
     private let edgeInsets: CGFloat = 28
+    private let listId: Int
     private var listSummaryHeight: CGFloat = 80
-    private var list: MediaList!
+    private var list: MediaList?
     private var sections = [Section]()
 
     private let headerReuseIdentifier = "HeaderReuseIdentifier"
     private let listSummaryCellReuseIdentifier = "ListSummaryCellReuseIdentifier"
     private let mediaCellReuseIdentifiter = "MediaCellReuseIdentifier"
 
+    init(listId: Int) {
+        self.listId = listId
+        super.init(nibName: nil, bundle: nil)
```

Comment:
should super.init come before self.listId? or does it give error cause listId is not given a value first

Reply:
yea I put it before bc otherwise I get an error